### ISSUE TITLE
[FlexibleColumnLayout]: Screen reader isn't able to read first column content

### ISSUE THREAD

**Issue by OliwiaGowor** [SAP/ui5-webcomponents#10747]
### Bug Description

When the layout in **FlexibleColumnLayout** is set to `OneColumn`, the screen reader cannot read content from the first column (find tables, headers on the page, focus on the content inside the column). It is probably caused by `aria-hidden` being set to true on  `startColumn` slot when the layout is set to `OneColumn`.

![Image](https://github.com/user-attachments/assets/48bd4caf-f231-4e16-b3e4-e14b10d82029)

### Affected Component

FlexibleColumnLayout

### Expected Behaviour

The screen reader should be able to access content inside the first column in **FlexibleColumnLayout**.

### Isolated Example

https://sap.github.io/ui5-webcomponents/components/fiori/FlexibleColumnLayout/

### Steps to Reproduce

1. Create FlexibleColumnLayout component 
2. Add some content to first column
3. Set layout to `OneColumn`
4. Try accessing the content using screen reader (or check the slot with inspect)

### Log Output, Stack Trace or Screenshots

<img width="448" alt="Image" src="https://github.com/user-attachments/assets/203418b5-ea6d-4626-8813-edd726ce76b6" />

<img width="1479" alt="Image" src="https://github.com/user-attachments/assets/36a5ab6c-9849-4f1b-b63a-2a3e97986030" />

### Priority

Medium

### UI5 Web Components Version

2.4.0/2.6.3

### Browser

Chrome

### Organization

Kyma

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by NHristov-sap** [comment]
Hi @OliwiaGowor ,

Can you please provide a live example that demonstrates the reported issue?

Best Regards,
Nikolay Hristov

**Comment by OliwiaGowor** [comment]
Hello @NHristov-sap ,

The provided Stackblitz is stuck on "Starting dev server" and it's not loading. There is also some problem with the playground for FlexibleColumnLayout in the documentation, so I'm not able to provide a live example

Best Regards, 
Oliwia Gowor

**Comment by NHristov-sap** [comment]
Hello @ui5-webcomponents-topic-p,

Please check if the FlexibleColumnLayout component behaves properly when there is only one column. Unfortunately the playground sample doesn't work for this component, so I (and the reporter) cannot give reproducible sample.

Best Regards,
Nikolay Hristov

**Comment by petyabegovska** [comment]
@NHristov-sap, is the problem reproducible on nightly: https://sap.github.io/ui5-webcomponents/nightly/components/fiori/FlexibleColumnLayout/?

**Comment by vladitasev** [comment]
How to reproduce:

Run the following HTML

```html
<ui5-flexible-column-layout layout="OneColumn">
	<div slot="startColumn">
		<ui5-list id="list">
			<ui5-li>Hello worild!</ui5-li>
			<ui5-li>Hello worild!</ui5-li>
			<ui5-li>Hello worild!</ui5-li>
			<ui5-li>Hello worild!</ui5-li>
		</ui5-list>
	</div>
</ui5-flexible-column-layout>
```

Check the DOM:

<img width="736" alt="Image" src="https://github.com/user-attachments/assets/c75b49e0-a585-473b-b3ca-5fc892169721" />
